Dockerfile.db

```diff
@@ -0,0 +1,8 @@
+FROM mysql
+
+ENV MYSQL_PASSWORD=zaphod \
+    MYSQL_USER=test \
+    MYSQL_DATABASE=test \
+    MYSQL_RANDOM_ROOT_PASSWORD=yes
+
+ADD storage/mysql/storage.sql /docker-entrypoint-initdb.d/storage.sql
```

Comment:
This is the Docker equivalent of `./script/resetdb` since this particular mysql image runs all the sql scripts in `/docker-entrypoint-initdb.d` when it's being initialized.